index.md

```diff
@@ -1,277 +1,71 @@
-> 
-> - [Overview of the dataset](http://www.datacarpentry.org/genomics-workshop/01-intro-to-dataset/)
+> This workshop uses data from a long term evolution experiment published in 2012: [Genomic analysis of a key innovation in an experimental Escherichia coli population](http://www.nature.com/nature/journal/v489/n7417/full/nature11514.html) by Blount ZD, Barrick JE, DAvidson CJ, and Lenski RE. (doi: 10.1038/nature11514)
+>
+> More information about these data will be presented in the [first lesson of the workshop](http://www.datacarpentry.org/organization-genomics/01-introduction/).
 {: .prereq} 
 
 # Workshop Overview
 
-This document provides basic information about Data Carpentry Genomics workshops for instructors:
-
-* general outline of a Genomics workshop 
-* location of materials
-* learning goals for each module and
-* instructor skills needed for each module.
-
-All of our materials are on GitHub with a [CC0](https://creativecommons.org/publicdomain/zero/1.0/) copyright waiver: [Data Carpentry curriculum on GitHub](https://github.com/datacarpentry)
+This page provides basic information about Data Carpentry Genomics workshops, including 
+a general outline of each of the two possible workshop tracks. Each workshop is designed to 
+be taught over two full days of instruction.
 
-<!--
-## Learning Objectives
+## Genomics Workshop with Pipeline Workflow
+This workshop includes introductions to project and data organization, 
+the command line, development and automation of 
+bioinformatics analysis pipelines, and the use of cloud computing.
 
-The following are the overarching learning objectives for the curriculum.
 
-* 
-* 
-* 
-* 
--->
+| Lesson    | Overview |
+| ------- | ---------- |
+| [Project organization and management](https://datacarpentry.github.io/organization-genomics/) | Learn how to structure your genomic metadata, understand read metadata returned from your sequencing facility, and access data on the NCBI short read archive (SRA) database.|
+| [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) |  Learn to navigate your file system, create, copy, move, and remove files and directories, and automate repetitive tasks using scripts and wildcards. |
+|[Data wrangling and processing](https://datacarpentry.github.io/wrangling-genomics/) | Use command-line tools to perform quality control, align reads to a reference genome, and identify and visualize between-sample variation. |
+|[Introduction to cloud computing for genomics](http://www.datacarpentry.org/cloud-genomics/) | Learn how to launch your own AWS instance and how to transfer data between your local computer and your virtual machine. |
```

Comment:
Should we put cloud computing after data organization, since that's the order in the workshop?

Reply:
From conversations with cloud computing maintainers, there was the idea to have the cloud computing lesson come at the end of the workshop. Learners will have used AWS (or some other remote system) for the previous lessons, but many want to continue to learn after the workshop and want to continue using AWS. The cloud computing lesson coming at the end can then teach learners how to create their own instances. @dvanic and @ACharbonneau - could you please confirm this understanding of the structure?

Reply:
That's not what I thought we were doing, but that doesn't mean it can't be. My thought was that the cloud part was quite near the beginning, because they need to get onto the cloud to do the rest of the lessons. 

Our lesson currently has the instructions for getting them onto the data carpentry instance, how to check that it's the right one, and a bit about how to move data onto and off the instance. 

We also have a page (or three, depending on how it turns out) about connecting and setting up an instance on your own at the end of the lesson.

Reply:
Thanks for weighing in @ACharbonneau. It looks like there are multiple different possible organizations for the cloud lesson. When I taught the Genomics lessons (the pipeline workflow version) this summer, we didn't do an explicit lesson on cloud computing, but just had the learners sign on through the shell without explaining what we were doing (much). We did have a lot of learners wanting to know how to reconnect after the workshop and how to set up their own instances in the future. To my mind, it makes more sense to have learners start with the shell, work through all of the tools that we teach there, and then learn about cloud computing at the end. At that point, they should be sufficiently motivated to want to continue learning about cloud/remote computing. We could even maybe demonstrate (or at least talk about) how much more difficult it would be for them all to do the analyses we've done in the first day and a half on their local computers (including installing all the software as well as the compute power/time it would take for the actual analyses). This could help further motivate the cloud lesson. 

I'm interested to hear thoughts from the other Genomics Maintainers. We'll also be discussing this at the meeting of the Genomics Maintainers group this week.

Reply:
Thanks @ACharbonneau that was my understanding too, and how I've been teaching it. 

I read through the shell lesson some more and see that in the shell lesson, we could have people log onto the Amazon instances and briefly discuss that they're connecting to cloud resources. Then at the end after they're excited about the command line and bioinformatics workflows, we could have a cloud lesson, that goes more into detail about what cloud resources are available and how you can move data from the cloud to your local computer. We do always get those types of questions, and they tend to come at the end, when people are now motivated to learn how to do this themselves. So, I think it would be great to put together the cloud lesson with this "next steps for using cloud computing" perspective. Having it at the end I think would make people more excited to learn it, and be there already when people do have those questions. 

I realize this is more of a 'cloud computing' discussion, so I'll go file an issue there about this.

Reply:
Issue submitted: https://github.com/datacarpentry/cloud-genomics/issues/35